geometry_msgs/Point[] point_array